src/main/webapp/user_profile.jsp

```diff
@@ -28,16 +28,15 @@
       <div id="user-name-container"><p>Name: <%=user.getUserName()%></p></div>
       <div id="user-year-container"><p>Graduation Year: <%=user.getUserYear()%></p></div>
       <div id="user-majors-container"><p>Majors: <%=user.getUserMajors()%></p></div>
-      <div id="user-skills-container"><p>Skills: <%=user.getUserSkills()%></p></div>
+      <div id="user-skills-container"><p>Top Skill: <%=user.getUserSkills()%></p></div>
       <div id="user-total-tasks-container">
           <p>Total Completed Tasks: <%=user.getUserTotal()%></p>
       </div>
       
       <div id="user-tasks-container">
         <h2><%=user.getUserName()%>'s Tasks:</h2>
         <%for (TaskData task : userTasks) {%>
-          <p>Task Name: <%=task.getName()%></p>
-          <p>Task Status: <%=task.getStatus()%></p>
+          <p><%=task.getName()%>, <%=task.getStatus()%></p>
         <%}%>
```

Comment:
You can the list-tasks.jsp file to display a list of tasks. See lines 55-56 in task.jsp to see how I included it in the task page.